picoCTF-web/api/team.py

```diff
@@ -132,7 +132,7 @@ def create_new_team_request(params, uid=None):
         team_name: The desired name for the team. Must be unique across users and teams.
         team_password: The team's password.
     Returns:
-        True if successful, exception thrown elsewise. 
+        True if successful, exception thrown elsewise.
```

Comment:
P.S. If we have other trailing spaces, I suggest we do a PR to clean them all as well.

Reply:
That seems to have been only one in the python code, though there are a good number in other parts of the repo.